pyramid_okta/request.py

```diff
@@ -0,0 +1,19 @@
+__author__ = 'psuresh'
+
+# from pyramid.decorator import reify
+
+class OktaRequestExtra(object):
+
+    def __init__(self, request):
+        self.request = request
+        self.session_token = None
```

Comment:
should call this `okta_session_token` or something, it is a little confusing otherwise